### PR TITLE
Add documentation for range.stride, range.alignment, range.aligned

### PR DESCRIPTION
**Pull request by daviditen**
Documentation was missing from the online chpldocs for these three methods.
Change them to secondary methods, and add comments for them to get them
documented.

Passed a full default paratest.
Did "make docs" and verified it looked OK.

### CODE REVIEW

**Comment by ben-albrecht** [comment]
If you wanted to knock out some other low-hanging fruit while in this file:

- document `proc low`
- document `proc high`
- Make `proc indexToStrideType` private instead of `pragma "no doc"`

File: modules/internal/ChapelRange.chpl [hunk 1]
```diff
@@ -621,8 +620,7 @@ proc _cast(type t, r: range(?)) where isRangeType(t) {
   //#
   
   // Moves the low bound of the range up to the next alignment point.
-  pragma "no doc"
-  /* private */ proc range.alignLow() 
+  private proc range.alignLow()
```
Comment (ben-albrecht) [inline modules/internal/ChapelRange.chpl]: Sorry I misled you on making these methods private:

```
error: Can't apply private to the fields or methods of a class or record yet
```

**Comment by daviditen** [comment]
Thanks for the review and suggestions.  No problem about "private", I've switched them back.  I'm doing another round of testing, then I think this will be set.

**Comment by daviditen** [comment]
[Summary: #Successes = 7234 | #Failures = 0 | #Futures = 0]

**Pull request merged**